home-page.css

```diff
@@ -0,0 +1,137 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.navigation-bar {
+    display: flex;
+    width: 100%;
+    background-color: rgb(150, 153, 153);
+    justify-content: space-between;
+    padding: 0.9rem;
+}
+.main-container{
+    position: sticky;
+    top: 0;
+    z-index: 1;
+}
+.title {
+    font-size: 2rem;
+    font-weight: 600;
+    margin-left: 1rem;
+    margin-top: 1.2rem;
+    color: rgb(4, 3, 8);
+
+}
+
+a{
+    text-decoration: none;
+}
+
+.search-bar {
+    width: 40%;
+    font-size: 1.2rem;
+    padding: 0.5rem;
+    margin: 1rem 0;
+}
+
+.side-nav-section {
+    display: flex;
+    width: 17rem;
+    justify-content: space-around;
+    align-items: center;
+    position: relative;
+
+}
+
+ul{
+    list-style: none;
+}
+
+li {
+    display: list-item;
+    text-align: -webkit-match-parent;
+}
+
+.login-btn {
+    font-size: 1.2rem;
+    padding: 0.5rem 0.8rem;
+}
+
+button {
+    cursor: pointer;
+}
+
+.badge-wishlist {
+    font-size: 1rem;
+    background-color: red;
+    color:white;
+    border-radius: 50%;
+    width: 1rem;
+    height: 1rem;
+    text-align: center;
+    position: absolute;
+    top: 0.8rem;
+}
+
+.btn {
+    font-size: 1.8rem;
+}
+
+
+.main-img{
+    width:100%;
+}
+.content{
+    position: absolute;
+    top: 50%;
+    left: 30%;
+    transform: translate(-50%,-50%);
+    font-size: 40px;
+}
+.shop-btn{
+    background-color: rgb(115, 115, 236);
+    position: absolute;
+    top: 60%;
+    left: 30%;
+    transform: translate(-50%,-50%);
+    font-size: 30px;
+    border: none;
+    border-radius: 50px;
+    padding: 11px;
+
+}
+.shop-btn:hover{
+    background-color: rgb(132, 132, 219);
+    color:black;
+}
+
+.features{
+    display: flex;
+    background-color:rgb(75, 73, 73);
+    width: 30rem;
+    padding: 0.5rem;
+    flex-grow: 1;
+    margin-left: 10px;
+}
+.arrivals {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.card-image{
+    height: 250px;
+    width:250px;
+
+}
+.arrivals{
+    background-color: rgb(192, 190, 189);
+    padding: 30px;
+    text-align: center;
+}
+.card_contianer{
+    display: flex;
+}   
```

Comment:
You could add an empty line at the end of the file to avoid such warnings by Github